app/main/views/feedback.py

```diff
@@ -0,0 +1,32 @@
+import requests
+
+from werkzeug.exceptions import ServiceUnavailable
+from werkzeug.datastructures import MultiDict
+from werkzeug.urls import url_parse
+
+from flask import current_app, request, redirect, flash, Markup
+
+from .. import main
+
+
+@main.route('/feedback', methods=["POST"])
+def send_feedback():
+    feedback_config = current_app.config['DM_FEEDBACK_FORM']
+    form_data = MultiDict()
+    for field, google_form_field in feedback_config['fields'].items():
+        form_data.setlist(google_form_field, request.form.getlist(field))
+
+    result = requests.post(feedback_config['uri'], list(form_data.items(multi=True)))
+    if result.status_code != 200:
+        raise ServiceUnavailable('Google forms submission problem (status %d)'.format(result.status_code))
+    came_from = url_parse(request.form['uri'])
+    # strip netloc and scheme as we should ignore attempts to make us redirect elsewhere
+    replaced = came_from._replace(scheme='', netloc='')
+
+    flash(Markup(
+        """Thank you for your message. If you have more extensive feedback, please
+        <a href="mailto:enquiries@digitalmarketplace.service.gov.uk">email us</a> or
+        <a href="https://airtable.com/shrkFM8L6Wfenzn5Q">take part in our research</a>.
```

Comment:
@karlchillmaid can you give this message (that the users gets if they send us feedback) a last once-over please?

As far as a I know the air-table request is still valid, although might change "soon-ish" I think!

Reply:
@galund @DilwoarH 

`If you have more detailed feedback, please email <a href="mailto:enquiries@digitalmarketplace.service.gov.uk">enquiries@digitalmarketplace.service.gov.uk</a> or <a href="https://airtable.com/shrkFM8L6Wfenzn5Q">take part in our research</a>.`

Can't vouch for how long the air-table request will be valid, though.

Reply:
@karlchillmaid - would it be better to have

`Thank you for your message. If you have more detailed feedback, please email <a href="mailto:enquiries@digitalmarketplace.service.gov.uk">enquiries@digitalmarketplace.service.gov.uk</a> or <a href="https://airtable.com/shrkFM8L6Wfenzn5Q">take part in our research</a>.`

Reply:
Yep, sorry – hadn't intended to drop the thank you part of the message, it's a nice touch. Was focused on the rest of the message and only posted the bit to change.

Reply:
How will we know if the airtable request expires (other than getting support requests about it)? We shouldn't use that link if it's liable to die on us with no warning.

Reply:
![screen shot 2017-08-29 at 15 50 28](https://user-images.githubusercontent.com/4599889/29827422-e4e15006-8cd1-11e7-92fd-0738d4a0f3bf.png)

@karlchillmaid - this is how it will look